abis_mapping/templates/incidental_occurrence_data_v3/mapping.py

```diff
@@ -3641,24 +3663,23 @@ def add_conservation_authority_collection(
 
     def add_sensitivity_category_attribute(
         self,
-        uri: rdflib.URIRef,
+        uri: rdflib.URIRef | None,
         row: frictionless.Row,
         dataset: rdflib.URIRef,
-        sensitivity_category_value: rdflib.URIRef,
+        sensitivity_category_value: rdflib.URIRef | None,
         graph: rdflib.Graph,
     ) -> None:
         """Adds Sensitivity Category Attribute to the Graph
 
         Args:
-            uri (rdflib.URIRef): URI to use for this node.
+            uri: URI to use for this node.
             row (frictionless.Row): Row to retrieve data from
             dataset (rdflib.URIRef): Dataset this belongs to
```

Comment:
Disagree if you want, but i think if we omit one arg type from a docstring then it should be omitted for all in that block. I'm basing this on the loose statement within the google guide of being "consistent".

Reply:
Yes good point. I was operating on "change least number of lines" mode. making this consistent for a function is sensible.